Content.Server/Ghost/GhostCommand.cs

```diff
@@ -23,13 +26,14 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 return;
             }
 
-            if (player.AttachedEntity is { Valid: true } frozen &&
-                _entities.HasComponent<AdminFrozenComponent>(frozen))
+            if (player.AttachedEntity is { Valid: true } attached &&
+                (_entities.HasComponent<AdminFrozenComponent>(attached) ||
+                !_cfg.GetCVar(CCVars.ICEnableGhost)))
```

Comment:
This really isn't the right place to put this check:

1. The denial message is "You cannot ghost right now.", which really does not clearly indicate that this is a server config thing.
2. This blocks using `ghost` while already dead, which is weird and actually breaks some mechanics that manually invoke the `ghost` command.